cli_tools/google-osconfig-agent/config/config.go

```diff
@@ -62,12 +63,24 @@ var (
 )
 
 type config struct {
-	osInventoryEnabled, osPackageEnabled, osPatchEnabled, osDebugEnabled                  bool
+	osInventoryEnabled, osPackageEnabled, osPatchEnabled, debugEnabled                    bool
 	svcEndpoint, googetRepoFilePath, zypperRepoFilePath, yumRepoFilePath, aptRepoFilePath string
 	numericProjectID, osConfigPollInterval                                                int
 	projectID, instanceZone, instanceName, instanceID                                     string
 }
 
+func (c *config) parsePreRelease(features string) {
+	for _, f := range strings.Split(features, ",") {
+		f = strings.TrimSpace(f)
+		switch f {
+		case "ospatch":
+			c.osPatchEnabled = true
+		case "ospackage":
+			c.osPackageEnabled = true
```

Comment:
With the current setup I can't log in this package as pretty much everything depends on it (including the logging package). If the existing run logs aren't enough we can look into other ways of logging these but with the current setup it would turn into a lot of log spam